source/renderer/app/api/api.js

```diff
@@ -583,6 +593,46 @@ export default class AdaApi {
     // }
   };
 
+  getAssets = async (request: GetAssetsRequest): Promise<GetAssetsResponse> => {
+    logger.debug('AdaApi::getAssets called', { parameters: request });
+    const { walletId } = request;
+
+    try {
+      const response = await getAssets(this.config, { walletId });
+
+      logger.debug('AdaApi::getAssets success', {
+        assets: response,
+      });
+      const assets = response.map((asset) => _createAssetFromServerData(asset));
+      return new Promise((resolve) =>
+        resolve({ assets, total: response.length })
+      );
+    } catch (error) {
+      logger.error('AdaApi::getAssets error', { error });
+      throw new ApiError(error);
+    }
+  };
+
+  getAsset = async (request: GetAssetRequest): Promise<GetAssetResponse> => {
```

Comment:
@DeeJayElly @daniloprates this one should never be used and it should be removed from the code.